python/llm/example/CPU/HF-Transformers-AutoModels/Advanced-Quantizations/AWQ/README.md

```diff
@@ -37,6 +41,17 @@ pip install accelerate==0.24.1
 pip install einops
 ```
 
+**(Optional) Support for LLM-AWQ Backend**
+
+BigDL-LLM uses `autoawq` as its default awq backend, and also automatically supports `llmawq` if your model is orginally quantized by llmawq. You can directly input an llmawq model in the same way as autoawq, after installing the following dependencies:
+
+```bash
+git clone https://github.com/mit-han-lab/llm-awq
```

Comment:
Do we really require users to install llm-awq? It seems nothing is imported from llm-awq in codes.

Reply:
You are correct, we have integrated the `llm-awq` code.